dimagi/utils/post.py

```diff
@@ -64,6 +65,22 @@ def simple_post(data, url, content_type="text/xml", timeout=60, headers=None, au
     return requests.post(url, data, **kwargs)
 
 
+def simple_xml_post(data, url, operation):
+    client = Client(url)
```

Comment:
This would expire in an hour. We can increase the time for that if needed but one hour seems good for now though i don't expect this to change much over time. Keeping it higher like a day would be safe as well.